az-digital/az_quickstart
# Closes #1710 Add new GH Actions workflow for preparing a minor release

## Description
Adding a github action for editing our repo in preparation for a minor release.

This action follows the guide here: https://github.com/az-digital/az_quickstart/wiki/Steps-to-create-a-minor-release

## Related issues
Closes #1710 
See Companion PR https://github.com/az-digital/az-quickstart-scaffolding/pull/123

## How to test
Go to https://github.com/az-digital/az_quickstart/actions/workflows/prepare-for-minor-release.yml
<img width="1312" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/c4c088bd-573a-4899-8a92-ab20ff0a1370">

Workflow looks like this:
<img width="1724" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/f9cee1cf-999a-4665-a095-5823802fd7ac">

See it here: https://github.com/trackleft/az_quickstart/actions/runs/5836839858

## Types of changes
CI

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
